primes.rb

```diff
@@ -1,6 +1,12 @@
+require 'prime'
+
 class Primes
-  def self.sum_to(limit = 100)
+  def self.sum_to(limit)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    primeArray = []
+	Prime.each(limit) do |prime|
```

Comment:
Try to keep your code consistently formatted and indented. It pays to keep your code styled according to the Ruby style guide (https://github.com/bbatsov/ruby-style-guide)

Try and pay particular attention to indentation as this helps other developers scan through your code and pick out individual blocks and elements easily.